pkg/config/vars.go

```diff
@@ -818,6 +818,14 @@ var configVars = map[string]configVar{
 			EnvVar: "ARTIFACT_HUB_REPO_ID",
 		},
 	},
+	"always-regenerate-chart-index": {
+		Type: boolType,
+		CLIFlag: cli.BoolFlag{
+			Name:   "always-regenerate-chart-index",
+			Usage:  "always regenerate the chart index and ignore the chart cache , it will cost the even more disk I/O",
```

Comment:
```suggestion
			Usage:  "always regenerate the chart index and ignore the chart cache (this will result in decreased performance and an increase in resource consumption)",
```